pkg/sql/logictest/logic.go

```diff
@@ -375,12 +375,15 @@ import (
 //  - sleep <duration>
 //    Introduces a sleep period. Example: sleep 2s
 //
-//  - user <username> [nodeidx=N]
+//  - user <username> [nodeidx=N] [forcenew]
```

Comment:
How would you feel about renaming this to `reconnect`, `re-establish`, `forcenewsession` or something of that nature? That seems a bit more inline with the documentation and provides some hinting about what the parameter does.

Reply:
Renamed it to `newsession`!